src/components/Badge/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+import { BaseProps } from '../types';
+import { StyledBadge, StyledBadgeContainer } from './styled/badge';
+
+type badgeVariant = 'default' | 'lightest' | 'outline-brand' | 'brand';
+
+interface Props extends BaseProps {
+    label?: ReactNode;
+    variant?: badgeVariant;
+    style?: object;
+}
+
+const Badge: React.FC<Props> = props => {
+    const { label, variant, style } = props;
+    return (
+        <StyledBadgeContainer>
```

Comment:
The container of all component should have the style prop passed, thus the consumer can customize the component.